simulator/docs/env-variables.md

```diff
@@ -0,0 +1,36 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator. 
+
+Please refer [docker-compose.yml](./docker-compose.yml) as an example
+use.
+
+`Port`: (required) This is the port number on which kube-scheduler-simulator
+server is started.
+
+`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
+etcd. The simulator runs kube-apiserver internally, and the
+kube-apiserver uses this etcd.
+
+`FrontendURL`: This URL represents the URL web UI started. The
```

Comment:
It's already renamed to CORS_ALLOWED_ORIGIN_LIST.

Reply:
Updated.